.github/workflows/images.yaml

```diff
@@ -264,8 +264,8 @@ jobs:
           IS_MERGE_GROUP: ${{ github.event_name == 'merge_group' }}
 
   manifests:
-    if: ${{ github.event_name == 'merge_group' && success('retina-images') && success('retina-win-images') && success('operator-images') && success('retina-shell-images')}}
     name: Generate Manifests
+    if: ${{ github.event_name == 'merge_group' }}
```

Comment:
does this mean that we will run this step regardless of whether the images were successfully built or not?

Reply:
Nope, per GH docs, the `needs` clause means a dependency must have succeeded, our success checks have always been redundant.
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#jobsjob_idneeds

Reply:
This PR does not change the issue that GH skips all downstream jobs if a dependency fails or is skipped, and re-run do not re-evaluate jobs with if-conditions. 
This has come up recently when WIndows MCR endpoints have been unstable.
I'm planning to an unconditional guard job to check if E2E has run.